app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly your `Log In` page has difference between [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), there are `Navbar` should doesn't appear and `Sign Up` button style should exact like [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) on `Log In` page. Please implment my suggestions 🙏🏼 

![image](https://github.com/YoutLone/Budget_App/assets/63915024/0c6011a4-69bb-4e1b-952d-6345ebe4c190)